src/components/NewsGroup.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { DateNewsCard,Button } from '@baltimorecounty/dotgov-components';
+const NewsGroup = (props) => {
+	const { Newsitems = [] } = props;
+	return (
+		<React.Fragment>
+            <div className="row">
+				{Newsitems.map((Newsitem) => (
+                    <div className="col col-lg-3">
```

Comment:
I slightly mispoke after testing this, but to save some time this should be `className="col-xl-3 col-lg-6"`

Reply:
done